statusHandler/termuic/termuiRenders/widgetsRender.go

```diff
@@ -3,10 +3,10 @@ package termuiRenders
 import (
```

Comment:
might move this file in statusHandler/views/termuic/termuiRenders package

Reply:
Done.